pkg/store/bucket.go

```diff
@@ -145,6 +146,11 @@ func newBucketStoreMetrics(reg prometheus.Registerer) *bucketStoreMetrics {
 		Name: "thanos_bucket_store_blocks_loaded",
 		Help: "Number of currently loaded blocks.",
 	})
+	m.lastLoadedBlock = promauto.With(reg).NewGauge(prometheus.GaugeOpts{
+		Name: "thanos_bucket_store_blocks_last_loaded",
+		Help: "Timestamp when last block got loaded.",
+	})
+	m.lastLoadedBlock.Set(0)
```

Comment:
I think people will get confused if we set this to `0` - on start up we will report that the last loaded block was at the unix epoch.

Reply:
I was in two minds :- start the clock from 1970 (or) start with lowest possible Unix timestamp. 

👍  Updated the code to start the initial timestamp to reflect Jan. 1st of 1970.